content/concepts/asset-manager.md

```diff
@@ -0,0 +1,2126 @@
+
+Here are some examples of a `single option` and `multiple options` attributes.
+![Examples of a single and a multiple options asset attributes](/img/concepts/single-and-multiple-options-asset-attributes.svg)
+
+And here is the JSON format of the `single option` attribute type.
+```json
+{
+  "code": "model_is_wearing_size",
+  "labels": {
+    "en_US": "Model is wearing size",
+    "fr_FR": "Le mannequin porte la taille"
+  },
+  "type": "single_option",
+  "value_per_locale": false,
+  "value_per_channel": false,
+  "is_required_for_completeness": true
+}
+```
+And here is the JSON format of the `multiple options` attribute type.
+```json
+{
+  "code": "main_colors",
+  "labels": {
+    "en_US": "Main colors",
+    "fr_FR": "Couleurs principales"
+  },
+  "type": "multiple_options",
+  "value_per_locale": false,
+  "value_per_channel": false,
+  "is_required_for_completeness": true
+}
+```
+
+### The `number` attribute
+The number attribute is useful to hold numeral information, such as a number of pages or a video duration.
+
+Here is an example of a `number` attribute.
+![An example of a number asset attribute](/img/concepts/number-asset-attribute.svg)
+
+And here is the JSON format of the `number` attribute type.
+```json
+{
+  "code": "number_of_pages",
+  "labels": {
+    "en_US": "Number of pages",
+    "fr_FR": "Nombre de pages"
+  },
+  "type": "number",
+  "value_per_locale": false,
+  "value_per_channel": false,
+  "is_required_for_completeness": true
+}
+```
+
+### The `media_file` attribute
+The media file attribute is useful to hold the binaries of a file, a file being a picture, a pdf, a video...
+
+Here is an example of a `media_file` attribute.
+![An example of a media file asset attribute](/img/concepts/media-file-asset-attribute.svg)
+
+And here is the JSON format of the `media_file` attribute type, for an image.
+```json
+{
+  "code": "picture",
+  "labels": {
+    "en_US": "Picture",
+    "fr_FR": "Photo"
+  },
+  "type": "media_file",
+  "media_type": "image",
+  "value_per_locale": false,
+  "value_per_channel": false,
+  "is_required_for_completeness": true,
+  "allowed_extensions": ["jpg"],
+  "max_file_size": "10"
+}
+```
+
+::: info
+As of the v3.2 and v4.0, you can only have images in this attribute. Indeed, the `media_type` field only accepts the `image` value.  
+In the next version, you will be able to add new types of files (like PDF).
+:::
+
+### The `media link` attribute
+The media link attribute type comes in pretty handy whenever you want your asset to refer to an external file, not hosted inside the PIM. For example, it can be very useful whenever you already have all your assets stored in a CDN or a DAM. The illustration below gives you an idea of how this attribute type can be used in such a case.
```

Comment:
```suggestion
The media link attribute type comes in pretty handy whenever you want your asset to refer to an external file, not hosted inside the PIM. For example, it can be very useful if you already have all your assets stored in a CDN or a DAM. The illustration below gives you an idea of how this attribute type can be used in such a case.
```